x/crosschain/types/message_vote_inbound.go

```diff
@@ -22,6 +22,13 @@ const MaxMessageLength = 10240
 // InboundVoteOption is a function that sets some option on the inbound vote message
 type InboundVoteOption func(*MsgVoteInbound)
 
+// WithMemoRevertOptions sets the revert options for inbound vote message
+func WithRevertOptions(revertOptions RevertOptions) InboundVoteOption {
+	return func(msg *MsgVoteInbound) {
+		msg.RevertOptions = revertOptions
+	}
```

Comment:
We can leave it as consistent for now and it will never be nill because the usage is very restricted. See [the usage](https://github.com/zeta-chain/node/blob/feat-btc-revert-address/x/crosschain/types/inbound_parsing.go#L146)